core/modules/field/modules/list/list.module

```diff
@@ -145,6 +144,29 @@ function list_field_settings_form($field, $instance, $has_data) {
   return $form;
 }
 
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+
+function list_form_field_ui_field_edit_form_alter(&$form, &$form_state) {
```

Comment:
Is it necessary to do this in a form_alter? Or should we change the original form instead?